src/components/button/_button.scss

```diff
@@ -241,14 +232,11 @@
   }
 
   .#{$prefix}--btn--ghost {
-    @include button-theme--x(
-      transparent,
-      transparent,
-      $interactive-03,
-      $ibm-colors__gray-10,
-      $interactive-03,
-      $ibm-colors__blue-70
-    );
+    @include button-theme--x(transparent, transparent, $interactive-01, $hover-ui, $interactive-01, $active-ui);
+    display: inline-flex;
```

Comment:
Is this specific to ghost button?

Reply:
yes the position of icon for ghost buttons with icons is slightly different from the other buttons in the spec